### DISCUSSION TITLE
Commits showing date information?

### DISCUSSION THREAD

**Discussion by iburunat** [magit/magit#4593] [Support] [answered]
Just a quick question: in the `magit-status` buffer, under `Recent commits`, is there a way to show the date of the commit (eg., on the side)? Thanks!

**Comment by kyleam** [comment, marked as answer]
Try calling `magit-toggle-margin` (bound to <kbd>LL</kbd> by default).

**Reply by iburunat** [reply]
Thanks, this indeed shows an extra column on the right showing the relative time, but not specific dates.

**Reply by kyleam** [reply]
You can use `magit-cycle-margin-style` (<kbd>Ll</kbd>) to cycle through different styles, and customize `magit-log-margin` to change the default style (including specifying an arbitrary string to feed to `format-time-string`).

https://magit.vc/manual/magit/Log-Margin.html

**Reply by iburunat** [reply]
Wonderful, thanks!
